### PR TITLE
[Cloud PC] Update description for Cloud PCs service plans

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

**Comment by RuiHou105** [comment]
**Roadmap workitem:**

Feature 11533: [Devices and apps][Cloud PC] Update description for Cloud PCs service plans

**Review by AshleyYangSZ — APPROVED** [review]

**Review by isvargasmsft — APPROVED** [review]

**Review by angelgolfer-ms — APPROVED** [review]
Hi @RuiHou105 , thanks for following up on these topics. I made some minor grammar fixes. Please take a look and let me know if you see any problems. Otherwise the PR is good to publish.

File: api-reference/beta/api/virtualendpoint-list-serviceplans.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-List the service plans that Cloud PCs support for a tenant.
+List the currently available service plans that an organization can purchase for their Cloud PCs.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/virtualendpoint-list-serviceplans.md:17]: On second thought, @RuiHou105 , when you say "currently available service plans", do you actually mean the plans listed on the marketing page?
I think letting customers see what an actual service plan is like helps make this API clearer.
Can you add the sentence that I suggested in our earlier email thread, in a separate paragraph after line 16:

"For examples of currently available service plans, see [Windows 365 compare plans and pricing](https://www.microsoft.com/en-us/windows-365/business/compare-plans-pricing). Currently, Microsoft Graph API is available for Windows 365 Enterprise."

Remember to update this paragraph when Windows 365 Business support the Microsoft Graph API as well.

File: api-reference/beta/resources/cloudpcserviceplan.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
-Represents a defined collection of service plan configurations that Cloud PCs support for a tenant.
+Represents a Windows 365 service plan configured for a Cloud PC.
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/cloudpcserviceplan.md:17]: @RuiHou105 , I suggest to add the similar info about service plans to this topic as well. I found looking at info for a service plan helpful to understand this API.
So, after line 17, can you add a new paragraph:

"For examples of currently available service plans, see [Windows 365 compare plans and pricing](https://www.microsoft.com/en-us/windows-365/business/compare-plans-pricing). Currently, Microsoft Graph API is available for Windows 365 Enterprise."

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@RuiHou105 , I suggest adding a link to the page that currently lists service plans, to help customers understand what a service plan for a Cloud PC is like, since virtual PCs is a relatively new service for some customers.

File: api-reference/beta/api/virtualendpoint-list-serviceplans.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-list-serviceplans.md:17]: Hi @angelgolfer-ms , yes the "currently available service plans" means the plans that are listed on the page and the tenant can buy for Cloud PCs.

I've added the sentence. Could you please help check whether the URL is correct? Thanks.

Comment (angelgolfer-ms) [reply api-reference/beta/api/virtualendpoint-list-serviceplans.md:17]: Thanks!
FYI I removed the language tag (en-us) segment in the HTTP link to the marketing site, for localization reasons. Please see this [topic](https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#locale-codes-in-external-links).

**Pull request merged**